Automattic/wp-calypso
# Help Center: Add tailored section mapping

This PR connects to the tailored articles backend changes done on D158372-code.
The tailored mapping are removed from the front-end.
To create a tailored section key that will match on the backend correctly a new mapping file was created: `tailored-section-mapping`. If the current route is connected to a tailored section it is sent to the backend, along with an empty query.


Fixes https://github.com/Automattic/dotcom-forge/issues/8502

## Proposed Changes

* Add tailored section mapping and send that to our endpoint

## Why are these changes being made?


* Keeping the article mapping on the front-end was preventing us from serving tailored articles across other locales without creating possible complex config.

## Testing Instructions
- Sync your sandbox with the latest trunk
- Checkout this branch and run calypso and also sync the help center app
- Test the tailored articles according to the tailored [sections defined](https://github.com/Automattic/wp-calypso/pull/93618/files#diff-3889c7f265a99203d6d76b224f47ac42dee677267aa23e16b333fd949c5ff04c)
- Test on different locales as well
- Ensure that the results returned are correct




## Pre-merge Checklist
